webapp/layouts/default.vue

```diff
@@ -146,6 +149,10 @@ export default {
       quickSearchResults: 'search/quickResults',
       quickSearchPending: 'search/quickPending'
     }),
+    userName() {
+      const { name } = this.user || {}
+      return name || this.$t('profile.userAnonym')
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-17T14:05:17Z" title="Wednesday, April 17th 2019, 4:05:17 pm +02:00">Apr 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -193,6 +196,16 @@ export default {
         return this.$route.path === url
       }
       return this.$route.path.indexOf(url) === 0
+    },
+    // TODO method this is a duplicate from /pages/profile/_id/_slug.vue
+    // where to put this?
```


@ulfgebhardt if you want to remove redundancy, VueJS has a couple of ways to do this. My suggestion would be a mixin. https://vuejs.org/v2/guide/mixins.html

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-19T10:02:55Z" title="Friday, April 19th 2019, 12:02:55 pm +02:00">Apr 19, 2019</time>_

---


Implemented as Mixin